lib/error.ex

```diff
@@ -16,12 +16,19 @@ end
 
 defimpl AshGraphql.Error, for: Ash.Error.Query.InvalidQuery do
   def to_error(error) do
+    error_fields =
+      if is_nil(error.field) do
+        Map.get(error, :fields, [])
+      else
+        [error.field]
+      end
```

Comment:
```suggestion
    fields = List.wrap(error.field || Map.get(error, :fields) || [])
```

There is a small difference here `Map.get(error, :fields) || []` vs `Map.get(error, :fields, [])` - second option would return `[]` for older Ash as there is no `fields` field, but will return (as I assume) nil for newer versions as there will be such key.

Reply:
Thanks for the suggestion 👍